Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Name the first stage so it can be reused later
+FROM rust:latest AS builder
+WORKDIR /app/revanced-discord-bot
+COPY . .
```

Comment:
Does this respect `.gitignore`? If not, you're going to have a **very bad** time.

Reply:
Yes, because it uses the git clone command to checkout the repo data, so whatever is ignored won't get pulled. Locally that's a whole different thing. Please suggest a change or even write it so we can merge.

Reply:
Yeah, I meant local. I figured that out the hard way, heh